src/wr_hsi_sensing/CMakeLists.txt

```diff
@@ -7,7 +7,13 @@ project(wr_hsi_sensing)
 ## Find catkin macros and libraries
 ## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
 ## is used, also find other catkin packages
-find_package(catkin REQUIRED COMPONENTS message_generation std_msgs rospy)
+find_package(catkin REQUIRED COMPONENTS 
+    message_generation
+    std_msgs
+    rospy
+    ublox_gps
+    mapviz
```

Comment:
I'm not sure if mapviz is needed here.

Reply:
Resolved in 3e4fac1a7d6007bafcf3fe0b1e02d20338dcb436